src/state/state.ts

```diff
@@ -13,6 +13,7 @@ export interface State {
   dataFetcherInterval?: NodeJS.Timeout;
   gasPriceStore: Record<string, Record<string, GasState>>;
   signedApiStore: Record<AirnodeAddress, Record<TemplateId, LocalSignedData>>;
+  signedApiUrlStore: Record<string, Record<AirnodeAddress, string>>;
```

Comment:
I've moved these to be available in the state, so they chould be stored here when batches are fetched. Let me know if you prefer something else, though.

Reply:
Seems good to me